wandb/cli/cli.py

```diff
@@ -1104,7 +1095,7 @@ def launch_sweep(
         entity=entity,
         obj_id=sweep_obj_id,  # if resuming
         launch_scheduler=launch_scheduler_with_queue,
-        state="PENDING",
+        state="PENDING" if not resume_id else "RUNNING",
```

Comment:
wait shouldn't it go back to pending if it's being queued again?

Reply:
Yeah actually you are totally right, we **always** want "PENDING". The only bit thats strange is if we are in a "PAUSED" state, we would then set it to pending, but thats fine I think.